### ISSUE TITLE
Please elaborate rules diagram

### ISSUE THREAD

**Issue by rushic24** [scikit-fuzzy/scikit-fuzzy#216]
Could you please explain further how to read the rule1.view() given here, Sorry this is not a place to ask this though.

**Comment by JDWarner** [comment]
The output is the actual Matplotlib `Figure` and `Axis` objects generated by NetworkX while visualizing a directed graph.

I would like to improve the visualization presented, and if you have ideas for doing so (color coding the various objects, labeling, improved overall layout) pull requests are quite welcome.

**Comment by MassEast** [comment]
I agree with both of you; the rule.view() does not make a rule clear at all and I also think that this should be improved. E.g., nodes should be labeled.
